packages/core-mobile/e2e/locators/collectibles.loc.ts

```diff
@@ -21,7 +21,7 @@ export default {
   nftlogo: 'avatar__logo_avatar',
   warningAddressRequired: 'Address required',
   warningInsufficientFee: 'Insufficient balance for fee.',
-  warningGasLimitIsInvalid: 'Gas limit is invalid',
+  warningGasLimitIsInvalid: 'Unable to send token, invalid gags limit.',
```

Comment:
you have a typo here is `gags` that should be `gas`

Reply:
Thank you for catching that! I've pushed the change.